### PR TITLE
Show associated social profiles on user profile page

### LINKED ISSUE #124: There's no list of linked accounts in the user profile
Based on #111

### PR DESCRIPTION
**Pull request by lucasrenan**
Show social media profiles associated to user account.

In this case, the associated profiles are `twitter` and `github`:
screen shot 2016-10-31 at 23 54 46

closes #124

### CODE REVIEW

**Comment by lucasrenan** [comment]
@lucianosousa @tauil @hugonomura review please ;)

File: app/views/profiles/show.html.erb [hunk 1]
```diff
@@ -9,14 +9,27 @@
...
+    <br>
     <ul>
       <% @profile.unassociated_providers.each do |provider| %>
         <li><%= link_to(provider.show_name, "auth/#{provider.name}") %></li>
       <% end %>
     </ul>
     <% end %>
-  </div>
+
+    <% unless @profile.authentications.empty? %>
```
Comment (lucianosousa) [inline app/views/profiles/show.html.erb]: `if @profile.authentications.any?` is not a better option here?
Also, can you put a print here?

Comment (lucasrenan) [inline app/views/profiles/show.html.erb]: all right!

**Comment by lucasrenan** [comment]
@lucianosousa description updated :)

**Comment by lucianosousa** [comment]
<3

**Comment by tauil** [comment]
:+1:

**Pull request merged**